tests/clients/test_psycopg2.py

```diff
@@ -19,7 +19,7 @@ class TestPsycoPG2(unittest.TestCase):
     def setUp(self):
         self.db = psycopg2.connect(host=testenv['postgresql_host'], port=testenv['postgresql_port'],
                                    user=testenv['postgresql_user'], password=testenv['postgresql_pw'],
-                                   database=testenv['postgresql_db'])
+                                   dbname=testenv['postgresql_db'])
```

Comment:
Now the scenario in which `db_parameter_name` is `database` and not `dbname` is not covered anymore, when it previously was. We should write variants to `test_basic_query` where we set up connections, with `dbname` `database` and `db` parameters as well.

Reply:
Well, I can do it, but as described in the `psycopg` [documentation](https://www.psycopg.org/docs/module.html#psycopg2.connect), `database` is a deprecated alias, and `dbname` is the connection parameter to the database name.